src/main/java/internetshop/controllers/cart/ShoppingCartController.java

```diff
@@ -14,15 +15,16 @@
 
 @WebServlet("/cart")
 public class ShoppingCartController extends HttpServlet {
```

Comment:
```suggestion
public class GetShoppingCartController extends HttpServlet {
```
давай трішки підправимо неймінг, щоб було зрозуміліше за що відповідає цей контроллер